megamek/src/megamek/common/weapons/WeaponHandler.java

```diff
@@ -557,6 +599,83 @@ protected int calcnClusterAero(Entity entityTarget) {
                         r.newlines = 0;
                         vPhaseReport.add(r);
                     } else {
+                        //If point defenses engage Large, single missiles
+                        if (pdBayEngagedMissile || amsBayEngagedMissile) {
+                            // remove the last reports because they showed the
+                            // number of shots that hit
+                            while (vPhaseReport.size() > reportSize) {
+                                vPhaseReport.remove(vPhaseReport.size() - 1);
+                            }
+                            int AMSHits = 0;
```

Comment:
prefer this declaration to be closer to the `for` loop

Reply:
Fixed I think